mm2src/coins/utxo/utxo_standard.rs

```diff
@@ -502,4 +503,15 @@ impl MmCoin for UtxoStandardCoin {
     fn swap_contract_address(&self) -> Option<BytesJson> { utxo_common::swap_contract_address() }
 
     fn mature_confirmations(&self) -> Option<u32> { Some(self.utxo_arc.conf.mature_confirmations) }
+
+    fn coin_protocol_info(&self) -> Option<Vec<u8>> {
+        Some(serde_json::to_vec(&self.utxo_arc.my_address.addr_format).unwrap())
```

Comment:
Please use `rmp_serde` here instead - it will provide more compact binary serialization.
As I can see, it can be also extracted to `utxo_common` and reused for other UTXO coins implementation.